padd.sh

```diff
@@ -145,6 +145,12 @@ TestAPIAvailability() {
             API_PORT=""
         else
             # API is available at this URL combination
+
+            if [ "${availabilityResonse}" = 200 ]; then
```

Comment:
The same typo identified by promofaux on the other PR:

```suggestion
            if [ "${availabilityResponse}" = 200 ]; then
```